src/pages/contributing.md

```diff
@@ -1,43 +1,200 @@
+
+There are other metadata fields like `title` and `description`, but those are typically parsed from the page itself.
+
+```md
+<!-- Second page in the sidebar -->
+---
+sidebar_position: 2
+---
+```
+
+#### Categories
+
+Categories are folders within the documentation. They inherit titles and positional data from `index.md`. If an `index.md` is not within a subfolder, a `_category_.json` file should be created, specifying the `label` representing the name of the section, and `position` representing where on the sidebar it should go.
+
+```js
+{
+    // Name of the category to display
+    "label": "Example Title",
+
+    // This will be rendered as the third element on the sidebar at the current level.
+    "position": 3
+}
+```
 
-Titles should be capitalized in the standard titling format. For example,
+### Titles
 
-* Guide For Contributing to This Documentation
-* Building and Testing Your Mod
+Titles are defined using up to six hashtags (`#`) to define each section. Titles should capitalize everything but unimportant words.
 
-Essentially, capitalize everything but unimportant words.
+```md
+<!-- There should only be one title with a single '#'. -->
+# Guide For Contributing to This Documentation
 
-Spelling, grammar, and syntax should follow those of American English. Also, prefer using separate words over contractions (e.g. "are not" instead of "aren't").
+### Building and Testing Your Mod
+```
+
+### Diction
+
+Spelling, grammar, and syntax should follow those in American English. Avoid using contractions in sentences; use two separate words ('is not' instead of 'isn't'). Additionally, avoid using pronouns when possible, unless you need to directly refer to the reader.
+
+### Paragraphs
+
+Paragraphs should be a continuous block, separated by a newline. Paragraphs should NOT have each sentence be on a new line.
+
+```md
+This is my first paragraph. See how the next sentence is on the same line? You can use word wrapping in your editor to stop the line from going off the screen.
+
+This is my next paragraph. It is separated by a new line.
+```
+
+### Indentation
+
+When indenting lines, use four spaces instead of tabs. Most markdown features require four spaces to recognize indentation, so it allows consistency across the document.
+
+```md
+* Hello World
+    * Four Spaces In
+```
+
+### Code References
+
+When referencing elements outside of code blocks, they should be surrounded with backticks (`` ` ``). Classes should use their simple name. Methods and fields should specify the class name followed by a `#`. If the class name is implied, the method or field can simply be prefixed with `#`. Inner classes should specify the name of the outer class followed by a `$`.
```

Comment:
I have some reservations with the use of `$` to represent inner classes. The dollar sign is used moreso in the _binary_ side of Java programming -- in the bytecode itself -- rather than in user-facing documentation. 

I would prefer using `.` to separate inner classes from their outer class, to be more intuitive to readers and make the separation between the two class names more distinct (insofar as a single `.` has more whitespace than `$`, leading to visual separation of the words).

Reply:
That's fine with me. I have no strong feelings either way.